launcher/net/Download.cpp

```diff
@@ -284,11 +331,11 @@ void Download::downloadReadyRead()
         auto data = m_reply->readAll();
         m_state = m_sink->write(data);
         if (m_state == State::Failed) {
-            qCritical() << "Failed to process response chunk";
+            qCCritical(DownloadLogC) << getUid().toString() << "Failed to process response chunk";
         }
         // qDebug() << "Download" << m_url.toString() << "gained" << data.size() << "bytes";
     } else {
-        qCritical() << "Cannot write download data! illegal status " << m_status;
+        qCCritical(DownloadLogC) << getUid().toString() << "Cannot write download data! illegal status " << m_status;
     }
 }
 
```

Comment:
Consider adding all of these changes to Upload.cpp as well :D

Reply:
It's in the plan, yes.